src/sentry/seer/similarity/similar_issues.py

```diff
@@ -40,6 +39,19 @@ def get_similarity_data_from_seer(
         ),
     )
 
+    if response.status > 200:
+        redirect = response.get_redirect_location()
+        if redirect:
+            logger.error(
+                f"Encountered redirect when calling Seer endpoint {SEER_SIMILAR_ISSUES_URL}. Please update `SEER_SIMILAR_ISSUES_URL` in `sentry.conf.server` to be '{redirect}'."  # noqa
+            )
+        else:
+            logger.error(
+                f"Received {response.status} when calling Seer endpoint {SEER_SIMILAR_ISSUES_URL}.",  # noqa
```

Comment:
lgtm, but just curious why `# noqa` was needed here?

Reply:
To avoid triggering the no-f-strings [lint rule](https://github.com/adamchainz/flake8-logging#log011-avoid-pre-formatting-log-messages). In principle I get it, but in this case I actually _want_ different statuses to group separately. (It occurs to me as I'm writing this that our parameterization might thwart me on that even so, but still...)